node/hyperbahn/service_proxy.js

```diff
@@ -225,6 +225,10 @@ function createServiceChannel(serviceName) {
         options.requestDefaults = self.serviceReqDefaults[serviceName];
     }
 
+    if (mode === 'exit') {
+        options.preferConnectionDirection = 'out';
```

Comment:
This is for service channels, i.e., egress nodes connecting to service processes.